react_components/BreakView.js

```diff
@@ -23,6 +23,7 @@ const BreakView = props => {
 };
 
 BreakView.propTypes = {
+  breakLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
   breakLabel: PropTypes.string.isRequired,
```

Comment:
Please remove this line.

Reply:
lol I forget it. I will take care tomorrow morning

Reply:
Fixed